Cargo.toml

```diff
@@ -13,3 +13,6 @@ members = [
     "sentry-panic",
     "sentry-types",
 ]
+
+[profile.dev.build-override]
+opt-level = 0
```

Comment:
m: This is for all devel builds?  why?

Reply:
https://doc.rust-lang.org/cargo/reference/profiles.html#overrides
I haven’t actually measured this, but its supposed to speed up building all the proc-macro stuff. On the other hand, I haven’t actually done any `--release` builds anyway, so I have no idea how slow we are for those.